lib/zone.ts

```diff
@@ -648,6 +651,17 @@ const Zone: ZoneType = (function(global: any) {
       return _currentTask;
     };
 
+    static __load_patch(name: string, fn: _PatchFn): void {
```

Comment:
I would prefer to keep this private to zone for now. Would love to see some use cases before we expose it.